### ISSUE TITLE
1. import libraries and load classifier

### ISSUE THREAD

**Issue by manasak2010** [manasak2010/number-plate-recognization#1]
``
import matplotlib.pyplot as plt
import numpy as np
import cv2
import tensorflow as tf
from sklearn.metrics import f1_score 
from tensorflow.keras import optimizers
from tensorflow.keras.models import Sequential
from tensorflow.keras.preprocessing.image import ImageDataGenerator
from tensorflow.keras.layers import Dense, Flatten, MaxPooling2D, Dropout, Conv2D

#Load the data required for detecting the license plates from cascade classifier.
plate_cascade = cv2.CascadeClassifier('C:/Users/KIIT/Desktop/plate detection/indian_license_plate.xml')
